### DISCUSSION TITLE
Color CSS hosted on CDN

### DISCUSSION THREAD

**Discussion by asbjornu** [picocss/pico#465] [Help] [answered]
It seems like the only way to refer to, say `pico.fluid.classless.purple.min.css`, is to download and self-host. It would be nice if the colored CSS files were also hosted on CDN so they could be referenced from there as well.

**Comment by lucaslarroche** [comment]
@asbjornu, all the versions are hosted on CDN:
https://cdn.jsdelivr.net/npm/@picocss/pico@2/css/pico.fluid.classless.purple.min.css

See here:
https://picocss.com/docs/version-picker/purple#usage-from-cdn

**Reply by asbjornu** [reply]
@lucaslarroche, yes. Sorry for being unclear. What I meant was that the CSS file defining all of the colors (`pico.colors.min.css`) seems to be unavailable on CDN. Please correct me if I'm wrong.

**Reply by lucaslarroche** [reply, marked as answer]
No problem 🙂

Here is the CDN file for the colors:
https://cdn.jsdelivr.net/npm/@picocss/pico@2/css/pico.colors.min.css

Here is the docs:
https://picocss.com/docs/colors#usage-with-css
